packages/buidler-core/test/internal/util/scripts-runner.ts

```diff
@@ -36,8 +36,12 @@ describe("Scripts runner", function() {
     );
     assert.equal(statusCode1, 0);
 
-    const statusCode2 = await runScript("./failing-script.js");
-    assert.notEqual(statusCode2, 0);
+    const statusCode2 = await runScript(
```

Comment:
Why was this change needed?

Reply:
I don't remember if the extra arguments were needed. The check for `123` was because at some point this tests was (wrongly) passing when the script failed  with a 1 exit code.

Reply:
Oh, I see. Good catch! Can we leave the `123` and remove the extra params if they aren't necessary?

Reply:
Done. It passes on my machine, will see if they continue to pass on windows (it would be extremely weird if they don't).